sotodlib/preprocess/preprocess_plot.py

```diff
@@ -253,14 +253,16 @@ def plot_pcabounds(aman, pca_aman, filename='./pca.png', signal=None, band=None)
 
     Parameters
     ----------
-    amans : AxisManager
+    aman : AxisManager
         input AxisManager
     pca_aman : AxisManager
         Relcal output AxisManager
-    ghz : str
-        Bandpass (for plotting purposes)
     filename : str
         Full filename with direct path to plot output directory.
+    signal : str
```

Comment:
Typically we have been trying to allow signal to be an array in addition to a string and use the following logic:

```
 if signal is None: 
     signal = aman['signal'] 
 elif isinstance(signal, str): 
     signal = aman[signal] 
 elif isinstance(signal, np.ndarray): 
     pass
 else: 
     raise TypeError("Signal must be None, str, or ndarray") 
```

So that the function can be used a bit more widely.